src/OpenColorIO/ops/fixedfunction/FixedFunctionOpGPU.cpp

```diff
@@ -523,6 +523,113 @@ void Add_LUV_TO_XYZ(GpuShaderCreatorRcPtr & shaderCreator, GpuShaderText & ss)
     ss.newLine() << pxl << ".rgb.g = Y;";
 }
 
+
+namespace 
+{
+    namespace ST_2084
+    {
+        static constexpr double m1 = 0.25 * 2610. / 4096.;
+        static constexpr double m2 = 128. * 2523. / 4096.;
+        static constexpr double c2 = 32. * 2413. / 4096.;
+        static constexpr double c3 = 32. * 2392. / 4096.;
+        static constexpr double c1 = c3 - c2 + 1.;
+    }
+
+    namespace HLG
+    {
+        static constexpr double E_MAX = 3.;
+
+        static constexpr double a = 0.17883277;
+        static constexpr double b = (1. - 4. * a) * E_MAX / 12.;
+        static const     double c0 = 0.5 - a * std::log(4. * a);
+        static const     double c = std::log(12. / E_MAX) * a + c0;
+        static constexpr double E_scale = 3. / E_MAX;
+        static constexpr double E_break = E_MAX / 12.;
+        static const     double Eprime_break = std::sqrt(E_break * E_scale);
+    }
+
+} // anonymous
+
+void Add_LINEAR_TO_PQ(GpuShaderCreatorRcPtr& shaderCreator, GpuShaderText& ss)
+{
+    using namespace ST_2084;
+    const std::string pxl(shaderCreator->getPixelName());
+
+    ss.newLine() << ss.float3Decl("sign3") << " = sign(" << pxl << ".rgb);";
+    ss.newLine() << ss.float3Decl("L") << " = abs(0.01 * " << pxl << ".rgb);";
+    ss.newLine() << ss.float3Decl("y") << " = pow(L, " << ss.float3Const(m1) << ");";
+    ss.newLine() << ss.float3Decl("ratpoly") << " = (" << ss.float3Const(c1) << " + " << c2 << " * y) / ("
+        << ss.float3Const(1.0) << " + " << c3 << " * y);";
```

Comment:
Please add a comment:
// The sign transfer here is very slightly different than in the CPU path, resulting in a PQ value of 0 at 0 rather than the true value of 0.836^78.84 = 7.36e-07, however, this is well below visual threshold.